source/Xunit.Gherkin.Quick.ProjectConsumer/App.config

```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
What makes me not comfortable about this is that this is XML... but I understand there may  not be another way.

Reply:
I've taken this from the xunit docs, but I'll see if I can find a better option, there must be a mechanism for updating app settings on nuget installation!

Reply:
So, its totally possible to do the configuration from a xunit.runner.json file. I've tried adding this to the contentFiles output for the nuget, but its not being copied over to the project. I'll take another look when I get a spare couple of hours :)